### PR TITLE
fix lazy loading for pagination

### PR DESCRIPTION
**Pull request by crissi**

### CODE REVIEW

**Comment by mfn** [comment]
LGTM 👍 , thanks!

**Comment by mfn** [comment]
@crissi ops, actually there was a bug in the travis scripts which swallowed the phpunit exit code in case it fails.

In short, this PR actually broke the lazyload_types=true build, see https://travis-ci.org/rebing/graphql-laravel/jobs/563293132#L440-L447

I only found this out by luck while removing the code coverage scripts in https://github.com/rebing/graphql-laravel/pull/429, as such I've reverted this PR.

Gladly accepting PRs fixing this with green tests!

**Comment by crissi** [comment]
Looks green to me. How do I run phpunit in both modes when developing?

Få Outlook til iOS<https://aka.ms/o0ukef>
________________________________
Fra: Markus Podar <notifications@github.com>
Sendt: Thursday, July 25, 2019 11:01:10 PM
Til: rebing/graphql-laravel <graphql-laravel@noreply.github.com>
Cc: Christian <c2n@live.dk>; Mention <mention@noreply.github.com>
Emne: Re: [rebing/graphql-laravel] fix lazy loading for pagination (#425)


@crissi<https://github.com/crissi> ops, actually there was a bug in the travis scripts which swallowed the phpunit exit code in case it fails.

In short, this PR actually broke the lazyload_types=true build, see https://travis-ci.org/rebing/graphql-laravel/jobs/563293132#L440-L447

I only found this out by luck while removing the code coverage scripts in #429<https://github.com/rebing/graphql-laravel/pull/429>, as such I've reverted this PR.

Gladly accepting PRs fixing this with green tests!

**Comment by mfn** [comment]
Only possible with setting the environment variable `TESTS_ENABLE_LAZYLOAD_TYPES=1`:
`TESTS_ENABLE_LAZYLOAD_TYPES=1 vendor/bin/phpunit`

That's how travis runs it.
